### ISSUE TITLE
How to use torch_geometric for Heterogeneous graph?

### ISSUE THREAD

**Issue by mrbeann** [rusty1s/pytorch_geometric#210]
## ❓ Questions & Help

I want to use torch_geometric for Heterogeneous graph and set only one kind of node with y values, but it seems I can't set some y with `None`. Any example of how to achieve this?  Thanks!

**Comment by rusty1s** [comment]
You should only save the values you want to train/test against in `y`, and additionally save a `train_idx` and `test_idx` in `data` (similar to training GCNs in semi-supervised setups).

**Comment by mrbeann** [comment]
Yes, I can do this. But what if in a heterogeneous graph, I only need to predict one kind of node(e.g., A) but do not have a label for kind B, C, etc. What can I do in this situation?

**Comment by rusty1s** [comment]
What I said above, e.g.:
```python
out = model(x, edge_index, ...)
loss = F.nll_loss(out[data.train_idx], data.y)
```
with `train_idx` saving the node indices of kind A.

**Comment by mrbeann** [comment]
Thanks, it works
